meteor/client/ui/RundownView.tsx

```diff
@@ -2889,6 +2888,27 @@ export const RundownView = translateWithTracker<IProps, IState, ITrackedProps>((
 									bucketDisplayFilter={this.props.bucketDisplayFilter}
 								/>
 							</ErrorBoundary>
+							<ErrorBoundary>
+								<SorensenContext.Consumer>
+									{(sorensen) =>
+										sorensen &&
+										this.props.studio &&
+										this.props.showStyleBase && (
+											<TriggersHandler
+												rundownPlaylistId={this.props.rundownPlaylistId}
+												showStyleBaseId={this.props.showStyleBase._id}
+												currentRundownId={this.props.currentRundown?._id || null}
+												currentPartId={this.props.currentPartInstance?.part._id || null}
+												nextPartId={this.props.nextPartInstance?.part._id || null}
+												currentSegmentPartIds={this.props.currentSegmentPartIds}
+												nextSegmentPartIds={this.props.nextSegmentPartIds}
+												sorensen={sorensen}
+												global={this.isHotkeyAllowed}
+											/>
+										)
+									}
+								</SorensenContext.Consumer>
+							</ErrorBoundary>
```

Comment:
Looks like an exact copy from above. Can we have this extracted to a method?

Reply:
It has been extracted to `renderSorensenContext`.